### ISSUE TITLE
Tasks: disappearing comment when editing a task

### ISSUE THREAD

**Issue by Cubone21** [influxdata/ui#97]
When editing a task and adding a comment into it, it won't save. It can save a comment only when creating a new task but after any kind of editing of the task the comment disappears.

![taskComment](https://user-images.githubusercontent.com/62006131/93986275-6c248880-fd86-11ea-91dc-81214062dcd3.gif)

**Comment by drdelambre** [comment]
seems to work now, reopen if its still broken

**Comment by genehynson** [comment]
Verified this is still broken

**Comment by genehynson** [comment]
After fixing this bug, make sure to re-enable this test: https://github.com/influxdata/ui/blob/master/cypress/e2e/shared/tasks.test.ts#L350

**Comment by kristinarobinson** [comment]
Blocked by https://app.zenhub.com/workspace/o/influxdata/idpe/issues/10368.
